### PR TITLE
Save runtime node as scene from remote scene tree.

### PR DESCRIPTION
**Pull request by Geequlim**

### CODE REVIEW

File: core/script_debugger_remote.cpp [hunk 1]
```diff
@@ -37,6 +37,7 @@
 #include "os/os.h"
 #include "project_settings.h"
 #include "scene/main/node.h"
+#include <scene/resources/packed_scene.h>
```
Comment (Noshyaar) [inline core/script_debugger_remote.cpp]: Shouldn't it be `" "` instead of `< >`?

Comment (Geequlim) [reply core/script_debugger_remote.cpp]: OK I will keep them the same format

Comment (akien-mga) [reply core/script_debugger_remote.cpp]: Yes, `< >` is for system headers.

**Comment by reduz** [comment]
Question,  this seems pretty useful, but why not able to save node and children besides just node?

**Comment by Geequlim** [comment]
@reduz  I use PackedScene to save the selected node. It does save children nodes of the selected one.

The only except is the root viewport node. Children nodes of the viewport cann't be saved I don't know why.

**Pull request merged**